cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -40,14 +40,14 @@ To use the **Docker image**:
 
 - Once Docker has started, pull down the *CDAP Docker Image* from the Docker hub using::
 
-    $ docker pull caskdata/cdap-standalone
+    $ docker pull caskdata/cdap-standalone:\ |literal-release|
```

Comment:
Right. My bad. Literal release can only be used in regular text. What you need is something different: a parsed literal:
```
.. container:: highlight

  .. parsed-literal::
  
    |$| docker pull caskdata/cdap-standalone:|literal-release|
```